src/oemof/tabular/facades/commodity_ghg.py

```diff
@@ -0,0 +1,211 @@
+import dataclasses
+
+from oemof.solph._plumbing import sequence
+from oemof.solph.flows import Flow
+from pyomo.core import BuildAction, Constraint
+from pyomo.core.base.block import ScalarBlock
+
+from oemof import solph
+
+from .commodity import Commodity
+
+
+@dataclasses.dataclass(unsafe_hash=False, frozen=False, eq=False)
```

Comment:
@SabineHaas - Is there a reason why you used this instead of @dataclass_facade?

Reply:
I opened a new issue for discussion - as I had this in my mind for quite a while:
https://github.com/oemof/oemof-tabular/issues/184